src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import countryReducer from './countries/countries';
+
+const reducer = combineReducers({
+  countries: countryReducer,
+});
+
+const store = createStore(reducer, applyMiddleware(logger, thunk));
+
```

Comment:
- [ ] Please do not log redux data on the browser console as it may cause site security issues. Kindly clear the logged data displayed on the browser console.